client/src/pages/guide/english/computer-hardware/cooling/index.md

```diff
@@ -20,5 +20,6 @@ Cooling devices are available for individual parts on the computer. The two most
 Room ventilation - Locate the computer where it is neutral to the rooms environmental registers. Keep the computer away from room heaters or vents at all times.
 * Room placement - Locate the computer away from windows or areas of strong sunlight exposure. Placement in closets can also provide challenges to cooling. It is recommended to keep the computer away from any carpet-like material, which can easily bring dust into the computer.
 * Placement next to other computers - Locating the computer away from other equipment allows air to flow in and out of the computer. Placement close to other equipment can constrict air flow and increase heat.
+* Cable Management = Having a rat's nest of cables can block airflow and make your computer hotter. It is reccomended to take some time to cable manage after building a computer, because it can increase the lifespan of your computer in the future.
```

Comment:
Use a dash instead of equals, like all the lines before it. Also it's r**ecomme**nded. Lifespan is always about the future, so you can just cut the sentece at "its lifespan".